pkg/detectors/falsepositives_test.go

```diff
@@ -173,7 +174,8 @@ func TestStringShannonEntropy(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if got := StringShannonEntropy(tt.args.input); got != tt.want {
+			got := roundToTwoDigits(StringShannonEntropy(tt.args.input))
```

Comment:
suggestion: I think we can rely on our testing package to handle most of the heavy lifting here

Ex:
```
got := StringShannonEntropy(tt.args.input)
if len(tt.args.input) > 0 && tt.want != 0 {
    assert.InEpsilon(t, tt.want, got, 0.1)
} else {
    assert.Equal(t, tt.want, got)
}
```

Reply:
That's cool! I didn't know about `InEpsilon`. Thanks for sharing. I updated the PR.

Reply:
I only just learned about it last week too. 😅